RANes/src/fds.cpp

```diff
@@ -614,9 +614,15 @@ static int SubLoad(FCEUFILE *fp) {
 		}
 
 		FCEU_fread(diskdata[x], 1, 65500, fp);
-		memcpy(&fds_ROM[x * 65500], diskdata[x], 65500);
-		md5_update(&md5, diskdata[x], 65500);
+
+		// Only copy the first set of subloaded data
+		if (!fds_ROM[x * 65500])
+		{
+			memcpy(&fds_ROM[x * 65500], diskdata[x], 65500);
+			md5_update(&md5, diskdata[x], 65500);
```

Comment:
`fds_ROM` appears to be solely used by us, so this change seems safe for that.

Regardless of whether or not it seems reasonable to not update the internal MD5, the current libretro core is still doing it, so it's probably best not to change that behavior.

https://github.com/libretro/libretro-fceumm/blob/master/src/fds.c#L588